modules/management-console-agent/src/main/java/org/apache/ignite/agent/StompDestinationsUtils.java

```diff
@@ -76,8 +76,8 @@ public static String buildSaveSpanDest(UUID clusterId) {
     /**
      * @return Metrics destination.
      */
-    public static String buildMetricsDest(UUID clusterId) {
-        return METRICS_DEST + clusterId + "/add";
+    public static String buildMetricsDest() {
+        return METRICS_DEST;
```

Comment:
@Klaster1 It is OK from fronrend side that `clusterId` not used any more in endpoint name?

Reply:
You mean all clusters will share the same metric definitions? That sounds wrong and totally not what we discussed before.

Reply:
@akuznetsov-gridgain Andrey remove cluster id from destination, because MetricResponse has cluterId in body.

Reply:
@akuznetsov-gridgain @Klaster1 This endpoint on backend where agent send metrics from nodes. This metrics package has binary format and already contains clusterId.

Reply:
Then I have no complaints.